addons/website_sale_delivery/models/sale_order.py

```diff
@@ -14,19 +14,22 @@ class SaleOrder(models.Model):
         compute='_compute_amount_delivery',
         string='Delivery Amount',
         help="The amount without tax.", store=True, tracking=True)
+    # VFE TODO this field help should be updated, bc the amount can be with tax depending
```

Comment:
![no_todo](https://user-images.githubusercontent.com/9530767/75356754-a7bea900-58b0-11ea-999d-8eaf1a7598ab.jpg)

Reply:
TODO for master (and with PO's advise), I do not want to update this in 13.0 :P.
The field is only shown in mail templates and the e-commerce so unless studio is used, it isn't shown as a field.

Reply:
create a task or a PR for master then :p